src/Nethermind/Nethermind.Synchronization/Blocks/BlockDownloadContext.cs

```diff
@@ -133,5 +133,14 @@ private void ValidateReceipts(Block block, TxReceipt[] blockReceipts)
                 throw new EthSyncException($"Wrong receipts root for downloaded block {block.ToString(Block.Format.Short)}.");
             }
         }
+
+        public void ForceDownloadReceipts()
```

Comment:
Maybe call it `SetDownloadReceipts`
`ForceDownloadReceipts` sounds like it will download directly.